readme.md

```diff
@@ -150,6 +150,13 @@ The `hevc` codec requires macOS 10.13 or newer. A computer with Intel 6th genera
 
 The [`proRes422` and `proRes4444`](https://documentation.apple.com/en/finalcutpro/professionalformatsandworkflows/index.html#chapter=10%26section=2%26tasks=true) codecs are uncompressed data. They will create huge files.
 
+#### destinationPath
+
+Type: `string`<br>
+Default: A `.mp4` file in a temporary directory
+
+Destination file path for the video.
+
```

Comment:
It should be noted that it's better to not set this and rather move the file when the export is done, so users don't see a incomplete file during recording.

Reply:
Or set this to a temp directory on a different disk if it's set for space reasons.